src/main/java/racingcar/game/RankService.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.game;
+
+import racingcar.car.Car;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class RankService {
+    private final HashMap<Integer, List<String>> ranks = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Integer, List<String>> ranks = new HashMap<>();
```
구현체 말고 Map 인터페이스를 사용하는 건 어떤가요?
코드가 유연해지고 확장성이 높아질 것 같습니다!